File: python/Quiz.py

print("Welcome to the Quiz Game")

score = 0
total = 3


print("What is the capital of France?")
print("1. Paris")
print("2. London")
print("3. Berlin")
print("4. Rome")
answer1 = int(input("Your answer : "))
if answer1 == 1:
    print("Correct")
    score += 1
else:
    print("Wrong")

print("Where is semicolon located?")
print("1. Ikeja")
print("2. Sabo Yaba")
print("3. Yaba Sabo")
print("4. Ikotun")
answer2 = int(input("Your answer : "))
if answer2 == 2:
    print("Nice")
    score += 1
else:
    print("Fair")

print("What is the color of Nigeria Flag?")
print("1. Green,white")
print("2. White,Green")
print("3. Green,Green")
print("4. Green,White,Green")
answer3 = int(input("Your answer : "))
if answer3 == 4:
    print("Good")
    score += 1
else:
    print("At this point I can't help you")

print("You scored", score, "out of", total,"")

File: java/Quiz.java
import java.util.Scanner;

public class QuizGame {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Welcome to the Quiz Game");

        int score = 0;
        int total = 3;

        System.out.println("What is the capital of France?");
        System.out.println("1. Paris");
        System.out.println("2. London");
        System.out.println("3. Berlin");
        System.out.println("4. Rome");
        System.out.print("Your answer : ");
        int answer1 = scanner.nextInt();

        if (answer1 == 1) {
            System.out.println("Correct");
            score += 1;
        } else {
            System.out.println("Wrong");
        }
	

        System.out.println("Where is semicolon located?");
        System.out.println("1. Ikeja");
        System.out.println("2. Sabo Yaba");
        System.out.println("3. Yaba Sabo");
        System.out.println("4. Ikotun");
        System.out.print("Your answer : ");
        int answer2 = scanner.nextInt();

        if (answer2 == 2) {
            System.out.println("Nice");
            score += 1;
        } else {
            System.out.println("Fair");
        }
	

        System.out.println("What is the color of Nigeria Flag?");
        System.out.println("1. Green,White");
        System.out.println("2. White,Green");
        System.out.println("3. Green,Green");
        System.out.println("4. Green,White,Green");
        System.out.print("Your answer : ");
        int answer3 = scanner.nextInt();

        if (answer3 == 4) {
            System.out.println("Good");
            score += 1;
        } else {
            System.out.println("At this point I can't help you");
        }
	

 	System.out.println("You scored " + score + " out of " + total);

    }



}


